src/busterbunny.js

```diff
@@ -49,6 +50,15 @@ module.exports = (function() {
             EVENT_NACKED: 'event-nacked'
         });
 
+        if(config.stats){
+            function emitStats()
+            {
+                self.emit(self.EVENTS.STATS, _stats);
```

Comment:
Correct, if you care to use the _stats object beyond this function, you should create at the very minimum a shallow copy before emitting.  If you care about data found for individual items in the array, then you would need to do a deep copy.